docs/Data/Lists/Head.md

```diff
@@ -0,0 +1,28 @@
+# `Head(list)`
+
+Extracts the first element of a non-null, non-empty list.
+
+## Arguments
+
+* `list (List<T>)`: The list.
+
+## Returns
+
+* `(T)`: First item in the list.
+
+## Throws
+
+* `ArgumentException` if the `list` is not a non-empty `List`.
```

Comment:
I'm of two minds regarding the "is not a non-empty `List`" here. I like that it's consistent with line `3`, but the double-negative causes the mind to do a double-take when reading. Consider "is null or empty".

Reply:
Hrm. I like you wording better.